lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -53,9 +57,20 @@ def get(self, request, course_key_string):
         course_key = CourseKey.from_string(course_key_string)
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
+        blocks = get_course_date_blocks(course, request.user, request, num_assignments=1, include_past_dates=False)
+
+        # User locale settings
+        user_timezone_locale = user_timezone_locale_prefs(request)
+        user_timezone = user_timezone_locale['user_timezone']
+
         data = {
             'course_tools': course_tools,
+            'course_date_blocks': [block for block in blocks if not isinstance(block, TodaysDate)],
+            'dates_tab_link': reverse('dates', args=[course.id]),
```

Comment:
This is going to end up being a little bit more complex, but we can talk through that. We are going to want to change the URL based on if the MFE is active for the dates tab.